Code/Source/HumanWorker/NpcNavigatorComponent.cpp

```diff
@@ -87,6 +91,13 @@ namespace ROS2::HumanWorker
         AZ::TickBus::Handler::BusDisconnect();
     }
 
+    bool NpcNavigatorComponent::IsClose(AZ::Vector3 vector1, AZ::Vector3 vector2, float acceptableDistanceError)
```

Comment:
Use `const &` when possible

Reply:
I agree, but this PR is just cherry-picks from the zpp_version branch. I will use const& in the next PR (https://github.com/RobotecAI/o3de-humanworker-gem/pull/8)